packages/framework/presence/src/latestMapValueManager.ts

```diff
@@ -191,7 +192,7 @@ class ValueMapImpl<T, K extends string | number> implements ValueMap<K, T> {
 		private readonly value: InternalTypes.MapValueState<T>,
 		private readonly localUpdate: (
 			updates: InternalTypes.MapValueState<T>,
-			forceUpdate: boolean,
+			options: LocalUpdateOptions,
 		) => void,
```

Comment:
I am eliminating this in my update. The value manager will have this. Shouldn't come from here. (bonus: eliminates a use of new type)